QUARKUS-711.md

```diff
@@ -0,0 +1,20 @@
+# QUARKUS-711 - RestEasy support for Multipart data
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-711
+
+Test plan focuses on RestEasy support for multipart data provided by `resteasy` and `resteasy-multipart` extensions, as described in:
+- https://quarkus.io/guides/rest-json#multipart-support
+- https://quarkus.io/guides/rest-client-multipart
+
+Apart from topics described below, the functionality remains unchanged.
+
+## Scope of the testing
+- Testing verifies:
+    - Correct multipart request data processing.
+    - The new default value of charset of parts of a multipart request is `UTF-8` (in contrast to previous `us-ascii`) if the encoding is not specified
+    - The value of the charset is overridden by property `quarkus.resteasy.multipart.input-part.default-charset`.
+    - The value of content type is specified by property `quarkus.resteasy.multipart.input-part.default-content-type`.
+- Multipart support extracted into a new `resteasy-multipart` extension - all affected applications must migrate by adding this extension.
```

Comment:
Where did you get this information?

To me https://github.com/quarkusio/quarkus/pull/11893 introduces new functionality

Reply:
Or did you mean `org.jboss.resteasy:resteasy-multipart-provider` dependency should be replaced by quarkus extension?

Reply:
Yes, that's what I meant. I'll rephrase that.